docs/en_US/install/3ds.md

```diff
@@ -21,12 +28,22 @@ Nimbus is available as both a 3DSX app and an installable CIA. The releases page
 
 Extract the contents of the zip archive to the root of your SD card. If you are asked to merge or overwrite files, accept the changes.
 
-You should now have the `0004013000002F02`, `0004013000003202`, and `0004013000003802` Luma patches along with the `nimbus.3dsx` and `nimbus.cia` homebrew.
+Ensure your SD card has all the following files
 
-<img src="/assets/images/docs/install/3ds/sd-card-luma.png" width=100% height=auto/>
-<img src="/assets/images/docs/install/3ds/sd-card-3ds.png" width=100% height=auto/>
+- `SD:/luma/titles/000400300000BC02/code.ips` (Miiverse, JPN)
+- `SD:/luma/titles/000400300000BD02/code.ips` (Miiverse, USA)
+- `SD:/luma/titles/000400300000BE02/code.ips` (Miiverse, EUR)
+- `SD:/luma/sysmodules/0004013000002F02.ips` (SSL)
+- `SD:/luma/sysmodules/0004013000003202.ips` (FRD (Friends))
```

Comment:
the double parentheses feels a little weird to me but I can live with it

Reply:
How about a comma, like with the Miiverse regions?

```suggestion
- `SD:/luma/sysmodules/0004013000003202.ips` (FRD, Friends)
```

Reply:
In my opinion if you're gonna change the separator I would make it a slash instead, since that's different than the regions (not really a sub-region thing)

Reply:
yeah, that looks better

Reply:
> In my opinion if you're gonna change the separator I would make it a slash instead, since that's different than the regions (not really a sub-region thing)

this is a better option

Reply:
I'll go with a slash as @hauntii suggested